src/test/java/org/jabref/logic/importer/fetcher/OpenAccessDoiTest.java

```diff
@@ -28,7 +28,7 @@ void setUp() {
     @Test
     void findByDOI() throws IOException {
         entry.setField(StandardField.DOI, "10.1038/nature12373");
-        assertEquals(Optional.of(new URL("https://dash.harvard.edu/bitstream/1/12285462/1/Nanometer-Scale%20Thermometry.pdf")), finder.findFullText(entry));
+        assertEquals(Optional.of(new URL("https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4221854")), finder.findFullText(entry));
```

Comment:
This should return a fulltext and should therefore end in pdf

Reply:
Check the expected value in https://github.com/JabRef/jabref/actions/runs/10771322038/job/29866598073.
The link does not lead to a pdf.

Reply:
So is there something wrong in `findFullText()`?

Reply:
Yes, that's what I was trying to say. findFullText should point to a pdf file that can be downloaded.
So the OpenAccessDOI fetcher neeeds to be adjusted

Reply:
Reverting the adapted change for `OpenAccessDoiFetcher` then for this PR.

Reply:
> Edit // https://unpaywall.org/data-format Seems like the fetcher is correct, ist just doesn't return a pdf anymore for that specific article

Let me find another article and adapt then.

Reply:
Weird. Still the same behavior.

Reply:
Okay, this time, it does link to a pdf, but the link doesn't end with `.pdf`.